apps/explorer/src/hooks/useResolveEns.ts

```diff
@@ -29,3 +30,15 @@ export function useResolveEns(address: string): AddressAccount | undefined {
 
   return addressAccount
 }
+
+async function resolveENS(name: string): Promise<string | null> {
```

Comment:
Move the function from another file because here is the only place where it is used